problem-2/Medium/Medium/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for Medium project.
+
+Generated by 'django-admin startproject' using Django 4.1.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-n6wk$(xot7_%4hz(+#wpet$&8lx#69d)x1db=$d&ezxj-dqp8y'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'blogs',
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
+
```

Comment:
```suggestion python
INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
     'blogs',
]

```

보통 first-party나 second-party app들은 import 순서가 관련있는 경우가 많아요. 저희가 만든 app을 제일 밑에 배정하는 편이 좋습니다.

Reply:
rest_framework가 installed_apps에 추가되지 않아서 관련 이슈가 있을 것 같아요.